### ISSUE TITLE
Fatal error on mach-composer update --check 

### ISSUE THREAD

**Issue by BramKaashoek** [labd/mach-composer#188]
Running on Ubuntu 20.04.4
mach-composer 2.2.0 (4b39c16) - 2022-06-10T07:56:32Z
git version 2.25.1

```
mach-composer update --check -f main.yml
Checking if there are updates for 17 components
fatal: option '--pretty=%H|%aN <%aE>|%ad|%s' must come before non-option arguments
```

**Comment by BramKaashoek** [comment]
Issue also occurs with git version 2.38.1

**Comment by mvantellingen** [comment]
Should be fixed with #189
